.github/workflows/ci-backend.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         id: meta
         uses: docker/metadata-action@v4
         with:
-          images: gran/gooleaver
+          images: backend
```

Comment:
Can we name the image: `gooleaver-backend`? To match the helm
![image](https://github.com/gran-software-solutions/gooleaver/assets/116893230/6e2d4d9b-3a7d-4eab-b419-ba34cb14f96d)